basics/installation/environments/windows-specific.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Installing an AMP stack on Windows
+menuTitle: Windows environment
+weight: 3
+---
+
+# Installing an AMP stack on Windows
+
+PrestaShop is an Open-Source web application running on an *AMP stack (Apache - MySQL - PHP).
+
+In order for PrestaShop to run locally on your Windows computer, you need to install an *AMP stack. 
+
+Several options are available, such as: 
+
+- [WAMP](https://www.wampserver.com/en/)
+- [XAMPP](https://www.apachefriends.org/index.html)
+- [EasyPHP](https://www.easyphp.org/easyphp-devserver.php)
+- [MAMP](https://www.mamp.info/en/mamp/windows/)
+- [Laragon](https://laragon.org/docs/)
+
+## Important notes when using Windows shell (CLI)
+
+Several examples in this documentation uses `CLI` (Command line interface), refered sometimes as `terminal`.
+
+You may need to replace some commands in examples, for example:
+
+- When copying files, on `Unix` based operating systems (the ones we use in the docs), we use `cp`, on Windows you need to use `copy`.
+- When moving files, on `Unix` based OS, we use `mv`, you need to use `move`. 
+- When deleting files, we use `rm`, you need to use `del`. 
+
+## Use WSL to run a AMP stack
+
+A good option to install an AMP stack on your Windows, is to use [WSL (Windows Subsystem for Linux)](https://learn.microsoft.com/en-gb/windows/wsl/install).
+
+`WSL` allows you to install a Linux distribution on your Windows, and leverage all Linux advantages when it comes to run a server, and it will help you understand how a webserver works, making it easier when you will have to deploy your PrestaShop for production. 
```

Comment:
```suggestion
`WSL` allows you to install a Linux distribution on your Windows, and leverage all Linux advantages when it comes to running a server, and it will help you understand how a web server works, making it easier when you will have to deploy your PrestaShop for production. 
```